features/admin.users.v1/components/wizard/add-user-wizard.tsx

```diff
@@ -141,6 +147,10 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
     const [ submitStep, setSubmitStep ] = useState<WizardStepsFormTypes>(undefined);
     const [ selectedGroupsList, setSelectedGroupList ] = useState<GroupsInterface[]>([]);
 
+    const isDistinctAttributeProfileForUserCreationDisabled: boolean = featureConfig?.users?.disabledFeatures?.includes(
```

Comment:
shall we use the new `useFeatureFlag` hook?

Reply:
`useFeatureFlag` mostly used to indicate the feature status whereas here the intention is to have a logic to disable feature if required. For feature disablement we are still utilizing the `disabledFeatures` array